### PR TITLE
Check in

### PR DESCRIPTION
**Pull request by ConConRob**
# Description
all elements for this view
can select a room and check in
can select a guest and checkout
only displays available rooms
only displays available guests
displays the passcode on checkin 
if invalid submit the ui gives feedback 

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
#31 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

- [ ] This change requires a documentation update

## Change status

- [ ] Complete, tested, ready to review and merge

- [x] Complete, but not tested (may need new tests)

- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Manually tested

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project

- [x] I have performed a self-review of my own code

- [ ] I have commented my code, particularly in hard-to-understand areas

- [x] I have made corresponding changes to the documentation, if relevant

- [x] I have run the app using my feature and ensured that no functionality is broken

- [x] My changes generate no new warnings

- [x] There are no merge conflicts

### CODE REVIEW

File: src/components/CheckOutForm.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      _id: propTypes.string.isRequired,
+      name: propTypes.string.isRequired,
+    }),
+  ).isRequired,
+  hotel_id: propTypes.string.isRequired,
+};
+
+const StyledCheckOutForm = styled.div``;
+
+export default CheckOutForm;
```
Comment (Pav0l) [inline src/components/CheckOutForm.js]: Do we have two files with `CheckOutForm.js` and `CheckoutForm.js` ? What is the difference there?

Comment (Pav0l) [reply src/components/CheckOutForm.js]: One of those should be renamed, as the naming is quite confusing.

File: src/store/actions/actionTypes.js [hunk 2]
```diff
@@ -54,22 +54,20 @@ export const UPDATE_HOTEL_SUCCESS = 'UPDATE_HOTEL_SUCCESS';
...
+  FETCH_ROOMS_FOR_HOTEL: 'FETCH_ROOMS_FOR_HOTEL',
+  FETCH_ROOMS_FOR_HOTEL_SUCCESS: 'FETCH_ROOMS_FOR_HOTEL_SUCCESS',
+  FETCH_ROOMS_FOR_HOTEL_FAILURE: 'FETCH_ROOMS_FOR_HOT',
+  UPDATE_ROOM_FOR_HOTEL: 'UPDATE_ROOM_FOR_HOTEL',
+  UPDATE_ROOM_FOR_HOTEL_SUCCESS: 'UPDATE_ROOM_FOR_HOTEL_SUCCESS',
+  UPDATE_ROOM_FOR_HOTEL_FAILURE: 'UPDATE_ROOM_FOR_HOT',
+  DELETE_ROOM_FOR_HOTEL: 'DELETE_ROOM_FOR_HOTEL',
+  DELETE_ROOM_FOR_HOTEL_SUCCESS: 'DELETE_ROOM_FOR_HOTEL_SUCCESS',
+  DELETE_ROOM_FOR_HOTEL_FAILURE: 'DELETE_ROOM_FOR_HOTEL_FAILURE',
+};
```
Comment (Pav0l) [inline src/store/actions/actionTypes.js]: well done organizing this into an object 🙌

File: src/components/CheckOutForm.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <option key={room._id} value={room._id}>
+              {room.name}
+            </option>
+          ))}
+        </select>
+        <input
+          placeholder='email'
+          onChange={event => this.setEmailInput(event.target.value)}
+        />
+        <button>Check Out</button>
```
Comment (Pav0l) [inline src/components/CheckOutForm.js]: What does this button do?

File: src/components/CheckOutForm.js (hunk 1 above)
Comment (mark-marshall) [reply src/components/CheckOutForm.js]: @ConConRob @Pav0l CheckoutForm is for billing but should be reserved for guest checkouts so I'll rename and then Conor can use whatever variation on it he wants

Comment (ConConRob) [reply src/components/CheckOutForm.js]: I had a CheckOutForm which I renamed to GuestCheckOutForm. This file didn't get removed when I renamed it so I'll make a push that fixed this. I think we should start putting our components in folders based on views.

File: src/components/CheckOutForm.js (hunk 3 above)
Comment (ConConRob) [reply src/components/CheckOutForm.js]: The PR from last night didnt have this part done

**Pull request merged**